FabiWare/keys.cpp

```diff
@@ -211,6 +212,26 @@ uint16_t getNextKeyName(char* keyNames, char* singleKeyName)
   return(i);
 }
 
+void storeKey(int k) {
+  for (int i=0;i<6;i++) {
```

Comment:
I think this is OK, because the value 6 comes from the HID keyboard report, which is hardcoded into Arduino's keyboard.cpp (and all other HID keyboard reports I know).

Reply:
OK, but if it would be a constant named like `HID_REPORT_KEY_COUNT` or similar this would be known by anyone who looks at the code ;)

Reply:
yes - this is the reason.
same for the BT-keyboard descriptor, right?
anyhow we could define a MAX_KEYS 6 constant.